### PR TITLE
refactor：smooth and simple new desktop

### PR DESCRIPTION
**Pull request by zjy365**
### Summary
🌐🛠️📚

This pull request updates the frontend desktop project by changing its name and version, removing unused files and scripts, reorganizing the project structure, and adding internationalization support for English and Chinese languages. The changes aim to improve the frontend code quality, performance, and user experience.

> _Sing, O Muse, of the glorious deeds of the desktop developers_
> _Who, with skill and wisdom, crafted the splendid frontend code_
> _And adorned it with many tongues, both English and Chinese_
> _To delight the eyes and ears of the users across the world._

### Walkthrough
* Change the project name and version in `package.json` ([link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-e096f3d2744612bf7e830137543c51d72c59489d165d7d230b8101bb65a84d85L2-R3))
* Remove unused or redundant scripts in `package.json` ([link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-e096f3d2744612bf7e830137543c51d72c59489d165d7d230b8101bb65a84d85L9-R49))
* Update the project dependencies in `package.json` and `README.md` ([link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-67e397a4dc2f9ce1d0b4f85017394904e082bfd71c6a2da59837975d5db36884L106-R137))
* Update the directory structure of the project in `README.md` and delete some unused or redundant directories and files ([link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-67e397a4dc2f9ce1d0b4f85017394904e082bfd71c6a2da59837975d5db36884L15-R84), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-71376650fd5028929566806c1cd25e4ba19b524af0abea42683931b9bbe6d7fa), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-afcd3f99eabd25c420a610f591955512170da879a56e9b102d1d6fa5153e3ef8), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-e4d97aae80d554c7c0abbe89e21bf6f8052e80f3b0297a4ffbd88908834fa724), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-4d5d5665299c53af4685e82a2d4fc56fb80ce0b46ed112cf27f3418dc83f4ccf), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-da174aeaa79dc20dbf58dc36fb4ce2548da22f34a36beaa973f9a9ddfec7090f), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-611893147b98c2c2a7cfa703d1200f2a1317e7d21ca82923731bfde1f57808b5), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-b923a385036f4708e4341141a128e45c3093cc83dce7472be183520d255d02cb), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-c966df0eadaee21f726f8e0e0bd31daba0c4ed02928415a3860f03599d758558), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-025224cfc97c6fc6a95bfce2c3a4e51eb81ad4f3844a8d65d3ee0b3ce2161883), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-0d52a8b35d27f497bd1a736f30ffd86b5474c603dd579ba61f6feeccd8e6525d), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-04c0a6a66858c1bf1c8564294eca33c77fda8adefc64ec55dbf0ea54ad0e054b))
* Add English and Chinese translations for some UI elements in `public/locales/en/translation.json` and `public/locales/zh/translation.json` ([link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-469b9b5a78e15a3f4d47da2694d3dd87938abdcafc7df2e7e5c37c32088638bdL1-R13), [link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-9360e8b98a4d6bc0e4f5b098d0d4cbd72e91f38400c3cb145f70d69d20be4f28L1-R21))
* Update the code reading instructions in `README.md` to reflect the new libraries and conventions ([link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-67e397a4dc2f9ce1d0b4f85017394904e082bfd71c6a2da59837975d5db36884L147-R153))
* Delete the `iconfont.js` file, as it is no longer needed ([link](https://github.com/labring/sealos/pull/3059/files?diff=unified&w=0#diff-7217e5769230d6b940abcc60086196f8ff1b705d93263959905e90003d4dd55dL1-L0))

### CODE REVIEW

File: frontend/desktop/src/components/floating_button/index.tsx
Comment (xudaotutou) [inline frontend/desktop/src/components/floating_button/index.tsx:1]: 有些注释没删

File: frontend/desktop/src/pages/api/desktop/getInstalledApps.ts
Comment (xudaotutou) [inline frontend/desktop/src/pages/api/desktop/getInstalledApps.ts:1]: 环境变量的注释没删

File: frontend/desktop/src/types/app.ts
Comment (xudaotutou) [inline frontend/desktop/src/types/app.ts:1]: 把app_store里的type整合一下

Comment (zjy365) [reply frontend/desktop/src/types/app.ts:1]: ok

File: frontend/desktop/src/stores/app.ts [hunk 1]
```diff
@@ -58,184 +60,71 @@ export type TAppConfig = {
...
-            state.installedApps = res?.data?.map((item: TApp, i: number) => {
-              return {
-                ...item,
-                ...initialFrantState,
-                // @ts-ignore nextline
-                order: state.getAppOrder(item)
-              };
-            });
+            state.installedApps = res?.data?.map((app: TApp) => new AppInfo(app, -1));
+            state.runner.loadApps(state.installedApps.map((app) => app.key));
```
Comment (c121914yu) [inline frontend/desktop/src/stores/app.ts:96]: 这里应该不用初始化吧？app没还运行

File: frontend/desktop/src/utils/delay.ts
Comment (c121914yu) [inline frontend/desktop/src/utils/delay.ts:1]: 合并到tools文件里？

File: frontend/desktop/src/stores/app.ts (hunk 1 above)
Comment (xudaotutou) [reply frontend/desktop/src/stores/app.ts:96]: 这里是装载app,用来收集app的key

File: frontend/desktop/deploy/Kubefile [hunk 2]
```diff
@@ -1,10 +0,0 @@
-FROM scratch
-USER 65532:65532
```
Comment (lingdie) [inline frontend/desktop/deploy/Kubefile:2]: please merge main codes and fix cluster image build.

File: frontend/desktop/deploy/README.md [hunk 3]
```diff
@@ -1,20 +0,0 @@
-### How to build image
-
-```shell
-sealos build -t docker.io/sealost/desktop:latest -f Kubefile .
-```
-
```
Comment (lingdie) [inline frontend/desktop/deploy/README.md:6]: This one too.

**Review by lingdie — CHANGES_REQUESTED** [review]

**Review by zzjin — APPROVED** [review]

**Review by lingdie — APPROVED** [review]

**Pull request merged**